src/shared/lib/pathwayMapper/PathwayMapperTable.tsx

```diff
@@ -31,6 +31,7 @@ interface IPathwayMapperTableProps {
     selectedPathway: string;
     changePathway: (pathway: string) => void;
     initialSortColumn?: string;
+    view: string;
```

Comment:
Ideally, this table should not now anything about the `view`, let's make it generic. Instead of passing the view let's just pass something like `columnsOverride`. See my comments below for details.